source/common/tls/ocsp/ocsp.cc

```diff
@@ -23,26 +23,27 @@ template <typename T> T unwrap(ParsingResult<T> res) {
 }
 
 unsigned parseTag(CBS& cbs) {
-  unsigned tag;
+  unsigned tag = 0;
   if (!CBS_get_any_asn1_element(&cbs, nullptr, &tag, nullptr)) {
-    throwEnvoyExceptionOrPanic("Failed to parse ASN.1 element tag");
+    IS_ENVOY_BUG("Failed to parse ASN.1 element tag");
```

Comment:
I don't think this is correct. I think invalid ASN.1 (from the ocsp staple data) could cause this.

However, this is only used to generate error strings when we're already in an error path. Maybe rename to `attemptParseTagForErrorMessage` and just have it return -1 in this case or something?